tests/Integration/Behaviour/Features/Context/Domain/Discount/DiscountFeatureContext.php

```diff
@@ -116,6 +117,26 @@ public function assertDiscount(string $discountReference, TableNode $tableNode):
         }
     }
 
+    /**
+     * @Then discount :discountReference shouldn't have the following properties:
```

Comment:
I didn't understand at first but with the scenario context it makes, I think it's worth adding some explanation here like

```suggestion
     * We need to check that fields were modified but we don't know the new value, only the old one (example: code)
     * @Then discount :discountReference shouldn't have the following properties:
```